app/src/main/java/ca/llamabagel/transpo/trips/domain/GetRefreshStateUseCase.kt

```diff
@@ -0,0 +1,19 @@
+package ca.llamabagel.transpo.trips.domain
+
+import ca.llamabagel.transpo.data.Result
+import ca.llamabagel.transpo.data.db.StopId
+import ca.llamabagel.transpo.trips.data.TripsRepository
+import kotlinx.coroutines.flow.Flow
+import kotlinx.coroutines.flow.asFlow
+import kotlinx.coroutines.flow.map
+import javax.inject.Inject
+
+class GetRefreshStateUseCase @Inject constructor(
+    private val repository: TripsRepository
+) {
+    suspend operator fun invoke(stopId: StopId): Flow<Boolean> {
+        return repository.getResultCache(stopId)
+            .asFlow()
```

Comment:
So here you use a regular body but in `GetNextBusTripsUseCase` you use an expression body... They're very similar functions shouldn't they both be expression bodies (or both be regular bodies)?